src/Campaigns/School/Footer.js

```diff
@@ -1,11 +1,126 @@
+import LinkedInIcon from "@mui/icons-material/LinkedIn";
+import InstagramIcon from "@mui/icons-material/Instagram";
+import { Container, Box } from "@mui/material";
+import TableFooter from "@mui/material/TableFooter";
+import Logo from "../../Assets/Logo/Logo.png";
+import { Link } from "react-router-dom";
 
-const Footer = () => {
+export const Footer = () => {
   return (
-    <div>
+    <TableFooter
+      sx={{
+        display: { xs: "flex" },
+        justifyContent: { xs: "center" },
+        alignItems: { xs: "center" },
+        justifyContent: { xs: "space-evenly" },
+        flexDirection: { xs: "column" },
+      }}
+    >
+      <Container
+        sx={{
+          display: { xs: "grid", sm: "flex" },
+          justifyContent: { xs: "center" },
+          alignItems: { xs: "center" },
+        }}
+      >
+        <Box
+          sx={{
+            display: { xs: "flex" },
+          }}
+        >
+          <img src={Logo} alt="Logo de la ONG" />
+        </Box>
 
-    </div>
+        <Box
+          sx={{
+            display: { xs: "flex" },
+            alignItems: { xs: "center" },
+            justifyContent: { xs: "space-evenly" },
+            width: { xs: "100%" },
+          }}
+        >
+          <Box
+            sx={{
+              display: { md: "flex" },
+              justifyContent: { xs: "center" },
+              alignItems: { xs: "center" },
+            }}
+          >
+            <Link to="">
+              <FacebookIcon sx={{ fontSize: 40 }} />
+            </Link>
+            <Box sx={{ display: { xs: "none", md: "flex" } }}>
+              <p>Facebook</p>
+            </Box>
+          </Box>
+
+          <Box
+            sx={{
+              display: { md: "flex" },
+              justifyContent: { xs: "center" },
+              alignItems: { xs: "center" },
+            }}
+          >
+            <Link to="">
+              <LinkedInIcon sx={{ fontSize: 40 }} />
+            </Link>
+            <Box sx={{ display: { xs: "none", md: "flex" } }}>
+              <p>Linkedin</p>
+            </Box>
+          </Box>
+
+          <Box
+            sx={{
+              display: { md: "flex" },
+              justifyContent: { xs: "center" },
+              alignItems: { xs: "center" },
+            }}
+          >
+            <Link to="">
+              <InstagramIcon sx={{ fontSize: 40 }} />
+            </Link>
+            <Box sx={{ display: { xs: "none", md: "inline-block" } }}>
+              <p>Instagram</p>
+            </Box>
+          </Box>
+
+          <Box
+            sx={{
+              display: { md: "flex" },
+              justifyContent: { xs: "center" },
+              alignItems: { xs: "center" },
+            }}
+          >
+            <Link to="">
+              <TwitterIcon sx={{ fontSize: 40 }} />
+            </Link>
+            <Box sx={{ display: { xs: "none", md: "inline-block" } }}>
+              <p>Twitter</p>
+            </Box>
+          </Box>
+        </Box>
```

Comment:
if you review this code its repeted for each social media, so, maybe you can create a function that render this info from a list and then its really easy to add more media:
```
const socialMediaIconStyle= { fontSize: 40 };
const socialMediaList = [
    { name: Instagram, icon: <InstagramIcon sx={socialMediaIconStyle} />}, 
    { ... }
]
const socialMediaBox = () => socialMediaList.map(media => (
 <Box
    sx={{
       display: { md: "flex" },
        justifyContent: { xs: "center" },
        alignItems: { xs: "center" },
      }}
 >
     <Link to="">{ media.icon }</Link>
     <Box sx={{ display: { xs: "none", md: "inline-block" } }}>
          <p>{ media.name }</p>
     </Box>
</Box>
));
```

Reply:
changes applied